test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -680,6 +680,210 @@ class MultipleDisposalsTriageControllerSpec
 
     }
 
+    "handling requests to display the country of residence page" must {
+
+      def performAction(): Future[Result] =
+        controller.countryOfResidence()(FakeRequest())
+
+      behave like redirectToStartWhenInvalidJourney(performAction, isValidJourney)
+
+      "display the page" in {
```

Comment:
you've also coded behaviour to redirect the user away from this page if they were a uk resident - that's not been tested though